<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_2"/>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Документы</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Соревнования</string>
     </property>
     <addaction name="actCompetitionViewer"/>
     <addaction name="actPrizeViewer"/>
     <addaction name="actParticipationViewer"/>
     <addaction name="actSponsorViewer"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Клубы</string>
     </property>
     <addaction name="actClubViewer"/>
     <addaction name="actClubCompetitionViewer"/>
     <addaction name="actClubSizeViewer"/>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Спортсмены</string>
     </property>
     <addaction name="actAthleteViewer"/>
     <addaction name="actCareerViewer"/>
    </widget>
    <addaction name="actFacilityViewer"/>
    <addaction name="menu_5"/>
    <addaction name="actSportViewer"/>
    <addaction name="actCoachViewer"/>
    <addaction name="menu_3"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actFacilityViewer">
   <property name="text">
    <string>Сооружения</string>
   </property>
  </action>
  <action name="actAthleteViewer">
   <property name="text">
    <string>Список спортсменов</string>
   </property>
  </action>
  <action name="actSportViewer">
   <property name="text">
    <string>Виды спорта</string>
   </property>
  </action>
  <action name="actCareerViewer">
   <property name="text">
    <string>Карьера</string>
   </property>
  </action>
  <action name="actCompetitionViewer">
   <property name="text">
    <string>Список соревнования</string>
   </property>
  </action>
  <action name="actCoachViewer">
   <property name="text">
    <string>Тренеры</string>
   </property>
  </action>
  <action name="actPrizeViewer">
   <property name="text">
    <string>Победители</string>
   </property>
  </action>
  <action name="actClubViewer">
   <property name="text">
    <string>Список клубов</string>
   </property>
  </action>
  <action name="actClubCompetitionViewer">
   <property name="text">
    <string>Соревнования</string>
   </property>
  </action>
  <action name="actParticipationViewer">
   <property name="text">
    <string>Участники</string>
   </property>
  </action>
  <action name="actClubSizeViewer">
   <property name="text">
    <string>Численность</string>
   </property>
  </action>
  <action name="actSponsorViewer">
   <property name="text">
    <string>Организаторы</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
